$OpenBSD: patch-include_xcb_compat_h,v 1.4 2013/05/16 18:42:30 dcoppa Exp $

Fix for our uncommon setup: up-to-date libxcb, but older xcb-util.

--- include/xcb_compat.h.orig	Mon May 13 17:59:30 2013
+++ include/xcb_compat.h	Mon May 13 18:01:03 2013
@@ -34,14 +34,4 @@
 #define xcb_icccm_wm_hints_get_urgency xcb_wm_hints_get_urgency
 #define xcb_icccm_get_wm_transient_for_from_reply xcb_get_wm_transient_for_from_reply
 
-#define XCB_ATOM_CARDINAL CARDINAL
-#define XCB_ATOM_WINDOW WINDOW
-#define XCB_ATOM_WM_TRANSIENT_FOR WM_TRANSIENT_FOR
-#define XCB_ATOM_WM_NAME WM_NAME
-#define XCB_ATOM_WM_CLASS WM_CLASS
-#define XCB_ATOM_WM_HINTS WM_HINTS
-#define XCB_ATOM_ATOM ATOM
-#define XCB_ATOM_WM_NORMAL_HINTS WM_NORMAL_HINTS
-#define XCB_ATOM_STRING STRING
-
 #endif
